test/test_tree_2d_eulermulti.jl

```diff
@@ -25,42 +33,86 @@ EXAMPLES_DIR = pkgdir(Trixi, "examples", "tree_2d_dgsem")
       l2   = [81.52845664909304, 2.5455678559421346, 63229.190712645846, 0.19929478404550321, 0.011068604228443425],
       linf = [249.21708417382013, 40.33299887640794, 174205.0118831558, 0.6881458768113586, 0.11274401158173972],
       initial_refinement_level = 3,
-      tspan = (0.0, 0.001),
-      output_directory="out")
-      lines = readlines("out/deviations.txt")
-      @test lines[1] == "# iter, simu_time, rho1_min, rho2_min"
-      @test startswith(lines[end], "1")
+      tspan = (0.0, 0.001))
+      # Ensure that we do not have excessive memory allocations 
+      # (e.g., from type instabilities) 
+      let 
+        t = sol.t[end] 
+        u_ode = sol.u[end] 
+        du_ode = similar(u_ode) 
+        @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 1000 
+      end
```

Comment:
This looks like it removed some stuff?

Reply:
That is due to some merge process that went wrong, well spotted!